app/javascript/controllers/contest_controller.js

```
@@ -6,11 +6,15 @@ export default class extends Controller {
     }
 
     connect() {
-        this.setCountDownTimer();
+        this._countdownId = this.setCountDownTimer();
```

ESLint

[error] Unexpected dangling '_' in '_countdownId'.

(no-underscore-dangle)